integration_tests/roborazzi/build.gradle

```diff
@@ -0,0 +1,57 @@
+import org.robolectric.gradle.AndroidProjectConfigPlugin
+
+apply plugin: 'com.android.library'
+apply plugin: AndroidProjectConfigPlugin
+apply plugin: 'kotlin-android'
+apply plugin: "com.diffplug.spotless"
+apply plugin: "io.github.takahirom.roborazzi"
+
+spotless {
+    kotlin {
+        target '**/*.kt'
+        ktfmt('0.42').googleStyle()
+    }
+}
+
+android {
+    compileSdk 34
+    namespace 'org.robolectric.integration.roborazzi'
+
+    defaultConfig {
+        minSdk 21
+        targetSdk 34
+    }
+
+
+    compileOptions {
+        sourceCompatibility = '1.8'
+        targetCompatibility = '1.8'
+    }
+
+    kotlinOptions {
+        jvmTarget = '1.8'
+    }
+
+    testOptions {
+        unitTests {
+            includeAndroidResources = true
+            all {
+                // For roborazzi users, Please use Roborazzi plugin and gradle.properties instead of this.
```

Comment:
Thanks for the comment.
The reason I wrote this is that I assumed Roborazzi users would examine this module and use this option in their projects as a template. However, it is preferable to use the property in gradle.properties because the Roborazzi Plugin accounts for this. In contrast, for Robolectric, it is slightly different. The property will be applied across all modules in Robolectric, and code readers may not consult gradle.properties since these are merely integration tests.
However, I'll revise the comment to avoid confusion.